### DISCUSSION TITLE
Exposing product.reviews to swelljs api

### DISCUSSION THREAD

**Discussion by y3owk1n** [swellstores/community#55] [Help] [answered]
Referring to this discussion, #27 , do we have any documentation that I can follow for the API permission modification? I would like to expose the API access for product.reviews. 

Also, is there anyway for us to request for more data from the product api for 'swell-js'? E.g. review_rating.

Thanks in advance!

**Comment by ericingram** [comment, marked as answer]
You can technically get access to any fields by adding custom permissions to a public API key. More here: https://developers.swell.is/guides/opening-content-permissions

And, it's possible to include any amount of permitted data in a single request by using `include` and `expand` parameters. It works the same way on the frontend API as it does on the backend, although again it has to fit key permissions. https://developers.swell.is/backend-api/querying/expanding
